app/src/pages/trace/TracePage.tsx

```diff
@@ -1270,64 +1283,66 @@ function LLMMessage({ message }: { message: AttributeMessage }) {
       padding="size-200"
       {...messageStyles}
     >
-      <Flex direction="column" alignItems="start">
-        <Text color="text-700" fontStyle="italic">
-          {role}
-          {message[MESSAGE_NAME] ? `: ${message[MESSAGE_NAME]}` : ""}
-        </Text>
-        {messageContent ? (
-          <pre
-            css={css`
-              text-wrap: wrap;
-              margin: var(--ac-global-dimension-static-size-100) 0;
-            `}
-          >
-            {message[MESSAGE_CONTENT]}
-          </pre>
-        ) : null}
-        {toolCalls.length > 0
-          ? toolCalls.map((toolCall, idx) => {
-              return (
-                <pre
-                  key={idx}
-                  css={css`
-                    text-wrap: wrap;
-                    margin: var(--ac-global-dimension-static-size-100) 0;
-                  `}
-                >
-                  {toolCall[TOOL_CALL_FUNCTION_NAME] as string}(
-                  {JSON.stringify(
-                    JSON.parse(
-                      toolCall[TOOL_CALL_FUNCTION_ARGUMENTS_JSON] as string
-                    ),
-                    null,
-                    2
-                  )}
-                  )
-                </pre>
-              );
-            })
-          : null}
-        {/*functionCall is deprecated and is superseded by toolCalls, so we don't expect both to be present*/}
-        {hasFunctionCall ? (
-          <pre
-            css={css`
-              text-wrap: wrap;
-              margin: var(--ac-global-dimension-static-size-100) 0;
-            `}
-          >
-            {message[MESSAGE_FUNCTION_CALL_NAME] as string}(
-            {JSON.stringify(
-              JSON.parse(
-                message[MESSAGE_FUNCTION_CALL_ARGUMENTS_JSON] as string
-              ),
-              null,
-              2
-            )}
-            )
-          </pre>
-        ) : null}
-      </Flex>
+      <CopyToClipboard text={messageContent}>
```

Comment:
this only copies the message content but want to keep the UI simple for now. Can switch this out to have a component per variant but this is simplest for now

Reply:
Clipboard undefined in Chrome. Works in Safari.

<img width="515" alt="Screenshot 2024-03-07 at 2 34 07 PM" src="https://github.com/Arize-ai/phoenix/assets/80478925/1663889a-3e43-499d-ad6b-2da9b4b6db0b">